app/services/queue_listener.rb

```diff
@@ -82,7 +82,8 @@ def create_payment(params)
   end
 
   def subscribe_member(params)
-    page_name = ENV['AK_SUBSCRIPTION_PAGE_NAME']
+    language = params[:params][:language].try(:upcase)
+    page_name = ENV["AK_SUBSCRIPTION_PAGE_#{language}"] || ENV['AK_SUBSCRIPTION_PAGE_EN']
     if page_name.blank?
       raise Error.new("Your ActionKit page for subscriptions from the home page has not been set!")
```

Comment:
Maybe as an extra bonus this could raise with `params[:params][:language]` so you know which environment variable you're missing